.github/workflows/build.yml

```diff
@@ -198,28 +161,20 @@ jobs:
         with:
           path: /tmp/images
           key: ${{ runner.os }}-images-${{ github.run_id }}-${{ github.run_number }}
+
       - name: Download Artifacts
         uses: actions/download-artifact@v3
         with:
           name: pacts
           path: ./pacts
-      - name: Retrieve pact
-        run: |
-          docker container create --name temp -v pacts_data:/root hello-world
-          docker cp ./pacts/ignored-ignored.json temp:/root/ignored-ignored.json
-          docker cp ./pacts/sirius-lpa-dashboard-sirius.json temp:/root/sirius-lpa-dashboard-sirius.json
+
       - name: Restore images
         run: |
           docker load -i /tmp/images/app.tar
           docker load -i /tmp/images/pact-stub.tar
-      - name: Start app
-        run: |
-          docker-compose -f docker/docker-compose.ci.yml up -d pact-stub app
 
-      - name: Cypress Tests
-        uses: cypress-io/github-action@v5.8.3
```

Comment:
We previously switched to this action because `cypress/included` was flakey. If it's fine now then that's great, but we might need to switch back if it recurs.